test/end2end/runner.ts

```diff
@@ -0,0 +1,22 @@
+const { chromium } = require('playwright');
+
+(async () => {
+  const browser = await chromium.launch();
+  const context = await browser.newContext();
+  const page = await context.newPage();
+
+  await page.goto('http://live-ol.local');
```

Comment:
Port `3000` is needed to me to fix https://github.com/ordilabs/live/pull/86#issuecomment-1514941652 

```bash
# test
npx playwright install
npx ts-node test/end2end/runner.ts
Test passed
```